src/mjolnir/landmark_builder.cc

```diff
@@ -183,6 +194,38 @@ void LandmarkDatabase::insert_landmark(const std::string& name,
   pimpl->vacuum_analyze = true;
 }
 
+Landmark LandmarkDatabase::get_landmark(const uint32_t pkey) {
```

Comment:
i wonder if we should allow batching, this could improve efficiency when we are fetching these things:

```suggestion
std::vector<Landmark> LandmarkDatabase::get_landmark(const std::vector<int64_t>& pkeys) {
```

Reply:
good idea. maybe we should allow both cases